drivers/gpu/drm/vc4/vc4_hdmi.c

```diff
@@ -3806,8 +3855,19 @@ static int vc4_hdmi_bind(struct device *dev, struct device *master, void *data)
 	return ret;
 }
 
+static void vc4_hdmi_unbind(struct device *dev, struct device *master, void *data)
+{
+	struct vc4_hdmi *vc4_hdmi = dev_get_drvdata(dev);
+
+	(void)master;
+	(void)data;
```

Comment:
Does any compiler actually complain over the unused parameters? 
Most of the unbind functions in vc4 don't use master or data, and yet don't worry about having to use all parameters.

Reply:
Good point, looks like linux disables unused warnings: https://www.spinics.net/lists/newbies/msg63816.html